### ISSUE TITLE
emqx server load high

### ISSUE THREAD

**Issue by memacs** [emqx/emqx#2868]
#### Environment

- OS: centos 7
- Erlang/OTP: R21/10.2.1
- EMQ: v3.1-rc.1

#### Description

the emqx server load suddenly become very high, the log is

```
2019-09-03 23:39:21.019 [error] bigim|ios|0AD72D1E-C3DA-FEFC-3EA2-B5F850BE1415@36.19.47.104:57824 crasher:
    initial call: emqx_connection:init/1
    pid: <0.1482.8186>
    registered_name: []
    exception error: no function clause matching 
                     ekka_locker:merge_results([{badrpc,
                                                 {'EXIT',
                                                  {badarg,
                                                   [{ets,insert_new,
                                                     [emqx_sm_locker,
                                                      {lock,
                                                       <<"bigim|ios|0AD72D1E-C3DA-FEFC-3EA2-B5F850BE1415">>,
                                                       <0.1482.8186>,1,
                                                       {1567,525160,985649}}],
                                                     []},
                                                    {ekka_locker,
                                                     acquire_lock,2,
                                                     [{file,
                                                       "src/ekka_locker.erl"},
                                                      {line,138}]},
                                                    {ekka_locker,
                                                     acquire_lock,3,
                                                     [{file,
                                                       "src/ekka_locker.erl"},
                                                      {line,124}]},
                                                    {rpc,
                                                     '-handle_call_call/6-fun-0-',
                                                     5,
                                                     [{file,"rpc.erl"},
                                                      {line,197}]}]}}}],
                                               [['emqx@l-emqx2.vc.cn1'],
                                                ['emqx@l-emqx10.vc.cn1']],
                                               []) (src/ekka_locker.erl, line 198)
      in function  ekka_locker:acquire_locks/4 (src/ekka_locker.erl, line 116)
      in call from emqx_sm_locker:trans/3 (src/emqx_sm_locker.erl, line 46)
      in call from emqx_protocol:try_open_session/2 (src/emqx_protocol.erl, line 718)
      in call from emqx_protocol:process/2 (src/emqx_protocol.erl, line 396)
      in call from emqx_connection:handle_packet/3 (src/emqx_connection.erl, line 417)
      in call from gen_statem:call_state_function/5 (gen_statem.erl, line 1660)
      in call from gen_statem:loop_event_state_function/6 (gen_statem.erl, line 1023)
    ancestors: [<0.2095.0>,<0.2094.0>,esockd_sup,<0.1913.0>]
    message_queue_len: 1
    messages: [{cancel_timer,#Ref<0.803463984.1066401799.254530>,14933}]
    links: [<0.2095.0>]
    dictionary: [{'$metrics',#{{counter,'bytes/received'} => 137}},
                  {'$logger_metadata$',
                      #{client_id =>
                            <<"bigim|ios|0AD72D1E-C3DA-FEFC-3EA2-B5F850BE1415">>,
                        peername => "36.19.47.104:57824"}},
                  {incoming_bytes,137},
                  {force_shutdown_policy,
                      #{max_heap_size => 0,message_queue_len => 0}}]
    trap_exit: false
    status: running
    heap_size: 17731
    stack_size: 27
    reductions: 731789
  neighbours:
```

**Comment by turtleDeng** [comment]
3.2.2 has been fixed, please use version 3.2.2.

**Comment by memacs** [comment]
thanks
